test/Microsoft.Azure.SignalR.Tests/ServiceEndpointProviderFacts.cs

```diff
@@ -106,8 +106,8 @@ internal void GetClientEndpointWithAppName(IServiceEndpointProvider provider, st
             Assert.Equal(expected, actual);
         }
 
-        [Fact]
-        internal void GenerateMutlipleAccessTokenShouldBeUnique()
+        [Fact(Skip = "Access token does not need to be unique")]
+        internal void GenerateMultipleAccessTokenShouldBeUnique()

```

Comment:
Why is this different in AspNet and Core?

Reply:
No its not, they are all skipped.

Reply:
Yes. Didn't see it.